src/touchstone/django/middleware.py

```diff
@@ -19,7 +19,10 @@ def process_view(
         view_args: Sequence[Any],
         view_kwargs: Mapping[str, Any],
     ) -> None:
-        if not hasattr(view_func, "view_class"):
-            return
+        if hasattr(view_func, "view_class"):
```

Comment:
As this keeps growing, whats your take on having django/celery/etc. getting wire into touchstone? What if we were to make something else that provides some features for django using touchstone. I am thinking in particular, do we update rover-django-utils with this middleware and touchstone? This way touchstone doesn't have to know about WHO is using this and then rover-django-utils, which already understands the tools involved, can solve the how do I connect touchstone and django?

Reply:
I think if specific users want something more powerful or more structured than what's provided here then totally. But I think since Django, DRF, and Django are so popular, at least some basic support in touchstone is good